docs/WritingTests.asciidoc

```diff
@@ -191,25 +191,29 @@ clone_job.pl --from localhost 42 _GROUP=0
 === Using snapshots to speed up development of tests
 [id="snapshots"]
 
-Sometimes it's annoying to run the full installation to adjust some
-test. It would be nice to have the VM jump to a certain point. +QEMU
-backend+ provides feature that allows a job to start from a snapshot that might
-help in this situation.
+Sometimes it is annoying to run the full installation to adjust some
+test. It would be nice to have the VM jump to a certain point. The +QEMU
+backend+ provides a feature that allows a job to start from a snapshot which
+can help in this situation.
 
 Depending on the use case, there are two options to help:
 
-* create and *preserve* snapshots for *every test* module run (+MAKETESTSNAPSHOTS+)
-  - offers more flexibility as test can be resumed almost at any point, however
-    disk space requirements are high (expect more than 30GB for one job)
-  - this mode is useful for fixing non fatal issues in tests and debugging SUT
+* Create and *preserve* snapshots for *every test* module run (+MAKETESTSNAPSHOTS+)
+  - Offers more flexibility as the test can be resumed almost at any point.
+    However disk space requirements are high (expect more than 30GB for one
```

Comment:
need to or fully remove certain digit or replace it with formula based on qcow size. For a job which tests mini CD it will be far away from 30 GB